### PR TITLE
#29 Add to index.html

### PR DESCRIPTION
**Pull request by rohinij15**

### CODE REVIEW

**Comment by mikelbrierly** [comment]
Thanks for getting your pull request in Rohini!! 

One thing I would note is usually branch names are [`kebab-case`](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) instead of [`camelCase`](https://en.wikipedia.org/wiki/Camel_case), So yours for example would be `rohini-change-to-index-html-code` (no dots or capital letters).

Don't worry about this PR, but just for next time! :smiley:

**Comment by mikelbrierly** [comment]
@Hanna-N9 If you want to go into the "Files changed" tab and leave a comment on the code, we'll be ready to merge this in!

**Review by Hanna-N9 — APPROVED** [review]
Looks good

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
     </head>
     <body>
         <h1>Mikel wuz here</h1>
+        <h4>Hey! Rohini wuz here!!</h4>
```
Comment (mikelbrierly) [inline index.html:11]: 👍 @Hanna-N9 @rohinij15 @aggarwalsshreya Just fyi this view in GitHub is where we will do our code review!

**Review by mikelbrierly — APPROVED** [review]

**Pull request merged**